server/test_factory.py

```diff
@@ -1,82 +0,0 @@
-# -*- coding: utf-8; -*-
-#
-# This file is part of Superdesk.
-#
-# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
-#
-# For the full copyright and license information, please see the
-# AUTHORS and LICENSE files distributed with this source code, or
-# at https://www.sourcefabric.org/superdesk/license
-
-from flask import json
-from superdesk.tests import TestCase, setup, setup_db_user, test_user, get_prefixed_url, add_to_context
-from app import get_app
-from unittest.mock import patch
-from settings import LDAP_SERVER
-from apps.ldap import ADAuth
-
-
-def setup_auth_user(context, user=None):
-    if LDAP_SERVER:
-        setup_ad_user(context, user)
-    else:
-        setup_db_user(context, user)
-
-
-def setup_ad_user(context, user):
-    """
-    Setup the AD user for the LDAP authentication.
-    The method patches the authenticate_and_fetch_profile method of the ADAuth class
-    :param context: test context
-    :param dict user: user
-    """
-    ad_user = user or test_user
-
-    '''
-    This is necessary as test_user is in Global scope and del doc['password'] removes the key from test_user and
-    for the next scenario, auth_data = json.dumps({'username': ad_user['username'], 'password': ad_user['password']})
-    will fail as password key is removed by del doc['password']
-    '''
```

Comment:
but yep, should change in superdesk-core [superdesk/tests/__init__.py#L204](https://github.com/superdesk/superdesk-core/blob/master/superdesk/tests/__init__.py#L204)